core/components/molecules/overlayHeader/__tests__/OverlayHeader.test.tsx

```diff
@@ -32,9 +33,41 @@ describe('OverlayHeader component', () => {
 
 describe('OverlayHeader component with props', () => {
 
-  it('renders subheading', () => {
-    const { getByTestId } = render(<OverlayHeader heading={Heading} onClose={onClose} subHeading={SubHeading} />);
+  it('renders heading and subheading', () => {
+    const { getByTestId, queryByTestId } = render(
+      <OverlayHeader heading={Heading} onClose={onClose} subHeading={SubHeading}/>
+    );
     expect(getByTestId('DesignSystem-OverlayHeader--Subheading')).toBeInTheDocument();
+    expect(getByTestId('DesignSystem-OverlayHeader--heading')).toBeInTheDocument();
+    expect(queryByTestId('DesignSystem-OverlayHeader--Button')).not.toBeInTheDocument();
```

Comment:
Also write tests for  text content inside `DesignSystem-OverlayHeader--Subheading` and `DesignSystem-OverlayHeader--heading`.